tests/python/relay/test_pass_partial_eval.py

```diff
@@ -73,17 +93,19 @@ def test_empty_ad():
     t = TensorType(shape, dtype)
     d = Var("d", t)
     f = Function([d], d)
-    g = dcpe(gradient(f))
+    g = dcpe(f, grad_mod='higher_order')
     expected = Function([d], Tuple([d, Tuple([op.ones_like(d)])]))
+    expected = transform.OptimizeOnExpr(expected, transform.InferType())
     assert alpha_equal(g, expected)
 
+
 def test_ad():
     shape = (10, 10)
     dtype = "float32"
     t = TensorType(shape, dtype)
     d = Var("d", t)
     f = Function([d], d * d)
-    g = dcpe(gradient(f))
+    g = dcpe(f, grad_mod='higher_order')
```

Comment:
there is no need to pass grad_mod around, it is really implicit.

Reply:
This is just to let the DCPE function know that gradient pass needs to be executed.